web-share/share-url-empty-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share with empty URL</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
```

Comment:
I was kinda expecting to see here, as a comment, or in the title, something about the conformance requirement. From this tests, it's not clear what the pass condition is when an empty URL is given.